styles/Datadog/words.yml

```diff
@@ -17,6 +17,7 @@ swap:
   'back(?:\s|-)end': backend
   'back(?:\s|-)ends': backends
   bear in mind: keep in mind
+  best in class: leading|top|excellent
```

Comment:
```suggestion
  'best(?:\s|-)in(?:\s|-)class': leading|top|excellent
```

Reply:
i did not think of that, but i like it! thanks.